easybuild/easyconfigs/l/libdeflate/libdeflate-1.7-GCCcore-9.3.0.eb

```
@@ -0,0 +1,38 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'ConfigureMake'
+
+name = 'libdeflate'
+version = '1.7'
+
+homepage = 'https://github.com/ebiggers/libdeflate'
+description = """Generic PCI access library."""
```

```suggestion
description = """Heavily optimized library for DEFLATE/zlib/gzip compression and decompression."""
```

This description is from other software. I've filed #12886 to correct this in the existing easyconfig.